README.md

```diff
@@ -1,5 +1,6 @@
-# REV-Underglow
+# REV3-Underglow
 
 ## Introduction
-
-*One-paragraph summary of the board and its purpose on the bike*
\ No newline at end of file
+Underglow is a board that will drive Adafruit Neopixel LED strips to provide underglow lighting to the bike. Each LED
+will have the ability to addressed individually by sending I2C data to a driver that controls the LED strips. Underglow
+will also be able to receive requests to change lighting configurations through CAN bus.
```

Comment:
Include the processor used for this board STM32F302. Also what is this I2C driver?